independent-publisher-connectors/Care Quaility Comission For England/apiProperties.json

```diff
@@ -0,0 +1,10 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Martyn Lesbirel",
+    "stackOwner": "Care Quality Commission",
+    "website": "https://anypoint.mulesoft.com/exchange/portals/care-quality-commission-5/"
```

Comment:
@martynlesbirel Please remove `website` property from here as it would throw build errors for us